react/index.js

```diff
@@ -535,8 +537,14 @@ EnhancedMinicart.schema = {
       type: 'string',
       isLayout: false,
     },
+    linkButtonFinishShopping: {
+      title: 'admin/editor.minicart.linkButtonFinishShopping.title',
+      type: 'string',
+      isLayout: false,
+    },
     labelButtonFinishShopping: {
       title: 'admin/editor.minicart.labelButtonFinishShopping.title',
+      default: '/checkout/#/cart',
```

Comment:
wrong, this should be above, in the `linkButtonFinishShopping`

Reply:
Changed. Thanks!